src/sql/parts/jobManagement/agent/agentView.component.ts

```diff
@@ -17,6 +17,8 @@ import { IBootstrapService, BOOTSTRAP_SERVICE_ID } from 'sql/services/bootstrap/
 import { DashboardServiceInterface } from 'sql/parts/dashboard/services/dashboardServiceInterface.service';
 import { AgentJobInfo, AgentJobHistoryInfo } from 'sqlops';
 import { PanelComponent, IPanelOptions, NavigationBarLayout } from 'sql/base/browser/ui/panel/panel.component';
+import { ControlHostContent } from 'sql/parts/dashboard/contents/controlHostContent.component';
```

Comment:
it seems like an inversion of the dependency diagram to have agent take a dependency on the host.  typically dependencies should be a layered DAG